buildshiprun/handler.go

```diff
@@ -75,12 +82,14 @@ func Handle(req []byte) string {
 		result, err := deployFunction(deploy, gatewayURL, c)
 
 		if err != nil {
+			reportStatus("failure", err.Error(), "DEPLOY")
 			log.Fatal(err.Error())
 		}
 
 		log.Println(result)
 	}
 
+	reportStatus("success", fmt.Sprintf("function successfully deployed as: %s", serviceValue), "DEPLOY")
```

Comment:
The status functionality must be put behind a feature flag so that we don't break the existing site or force everyone to use status updates. Not everyone will want this feature.

For instance:

```
report_status: "true"
```